.gitignore

```diff
@@ -2,3 +2,4 @@
 .cproject
 /build
 
+/Debug/
```

Comment:
Also I tend to update and not using current PPSDK in the future, if we are going to ignore `Debug` directory, could we also ignore `Release` which is also generated by Eclipse?

Reply:
Yes, I'll add it to ignore list later